contracts/src/v0.8/operatorforwarder/dev/AuthorizedForwarder.sol

```diff
@@ -28,7 +28,7 @@ contract AuthorizedForwarder is ConfirmedOwnerWithProposal, AuthorizedReceiver {
   }
 
   // solhint-disable-next-line chainlink-solidity/all-caps-constant-storage-variables
-  string public constant typeAndVersion = "AuthorizedForwarder 1.0.0";
+  string public constant typeAndVersion = "AuthorizedForwarder 1.1.0";
```

Comment:
forgot to bump in previous PR, adding a new method/upgrading version constitutes a version bump imho